src/azure-cli/azure/cli/command_modules/mysql/tests/latest/recordings/test_mysql_flexible_server_proxy_resource.yaml

```diff
@@ -2544,7 +2544,7 @@ interactions:
         what happens when the server cannot write to the binary log, which can cause
         the master''s log to become inconsistent and replication slaves to lose synchronization.","defaultValue":"ABORT_SERVER","dataType":"Enumeration","allowedValues":"IGNORE_ERROR,ABORT_SERVER","source":"system-default","isConfigPendingRestart":"False","isDynamicConfig":"True","isReadOnly":"True","documentationLink":"https://dev.mysql.com/doc/refman/5.7/en/replication-options-binary-log.html#sysvar_binlog_error_action"},"id":"/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/clitest.rg000001/providers/Microsoft.DBforMySQL/flexibleServers/azuredbclitest-000002/configurations/binlog_error_action","name":"binlog_error_action","type":"Microsoft.DBforMySQL/flexibleServers/configurations"},{"properties":{"value":"0","currentValue":"0","description":"Number
-        of seconds to service waits before the binary log file gets purged. See: https://docs.microsoft.com/en-us/azure/mysql/flexible-server/concepts-server-parameters#binlog_expire_logs_seconds","defaultValue":"0","dataType":"Integer","allowedValues":"0-4294967295","source":"system-default","isConfigPendingRestart":"False","isDynamicConfig":"True","isReadOnly":"False","documentationLink":"https://docs.microsoft.com/en-us/azure/mysql/concepts-server-parameters#binlog_expire_logs_seconds"},"id":"/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/clitest.rg000001/providers/Microsoft.DBforMySQL/flexibleServers/azuredbclitest-000002/configurations/binlog_expire_logs_seconds","name":"binlog_expire_logs_seconds","type":"Microsoft.DBforMySQL/flexibleServers/configurations"},{"properties":{"value":"ROW","currentValue":"ROW","description":"This
+        of seconds to service waits before the binary log file gets purged. See: https://learn.microsoft.com/azure/mysql/flexible-server/concepts-server-parameters#binlog_expire_logs_seconds","defaultValue":"0","dataType":"Integer","allowedValues":"0-4294967295","source":"system-default","isConfigPendingRestart":"False","isDynamicConfig":"True","isReadOnly":"False","documentationLink":"https://learn.microsoft.com/azure/mysql/concepts-server-parameters#binlog_expire_logs_seconds"},"id":"/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/clitest.rg000001/providers/Microsoft.DBforMySQL/flexibleServers/azuredbclitest-000002/configurations/binlog_expire_logs_seconds","name":"binlog_expire_logs_seconds","type":"Microsoft.DBforMySQL/flexibleServers/configurations"},{"properties":{"value":"ROW","currentValue":"ROW","description":"This
```

Comment:
We don't need to change files in `recordings` folder. They are recordings.

Reply:
@bebound, okay, I will take them out.  That will fix all of these errors?  -->

![image](https://github.com/Azure/azure-cli/assets/58762114/1af20cd7-1e08-40b4-a8cc-81dd893d41b1)